core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualificationArgs.scala

```diff
@@ -152,6 +152,9 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
         "databricks-azure. " +
         "Default is onprem.",
       default = Some("onprem"))
+  val speedupFactorFile: ScallopOption[String] =
+    opt[String](required = false,
+      descr = "Custom speed up factor file")
```

Comment:
Can we have similar description in README and Scallop?

Reply:
Thank you @parthosa! Changes updated.